services/github/github-languages-base.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const Joi = require('joi')
+const { GithubAuthService } = require('./github-auth-service')
+const { errorMessagesFor } = require('./github-helpers')
+
+/*
+Only validate the response is an object because we're expecting a response like
+{ "Python": 39624, "Shell": 104 }
+The keys could be anything and {} is a valid response (e.g: for an empty repo)
+*/
+const schema = Joi.object().required()
```

Comment:
I think this could use something like `Joi.object().pattern(/./, Joi.number().integer().required())`.